src/apis/mime_type_util.py

```diff
@@ -11,20 +11,19 @@ class MimeType(Enum):
     N3 = "text/n3"
     JSON = "application/json"
 
-    def to_ld_format(self) -> Optional[str]:
-        if self == MimeType.JSON_LD:
+    def to_ld_format(self) -> str:
+        if self is MimeType.JSON_LD:
             return "json-ld"
-        elif self == MimeType.RDF_XML:
+        elif self is MimeType.RDF_XML:
             return "xml"
-        elif self == MimeType.TURTLE:
+        elif self is MimeType.TURTLE:
             return "ttl"
-        elif self == MimeType.N_TRIPLES:
+        elif self is MimeType.N_TRIPLES:
             return "nt11"
-        elif self == MimeType.N3:
+        elif self is MimeType.N3:
             return "n3"
-        elif self == MimeType.JSON:
+        elif self is MimeType.JSON:
             return "json-ld"
```

Comment:
Thanks for the documentation link. Was struggling with Enums before, so it's good to get some more insight. knowing that identity instead of equality operator is recommended is helpful.

Reply:
So it's still a comparison of values, but some options have the same value and are equivalent.. Can be helpful when you want to indicate that some options are actually equivalent, although they have different appearances.

Reply:
we're not done yet with the enums :-{

Reply:
I think it is possible to enforce unique values using a decorator: https://docs.python.org/3/library/enum.html#ensuring-unique-enumeration-values

Reply:
or use auto values: https://docs.python.org/3/library/enum.html#using-automatic-values